File: uri/1 - Basics/1008/Main.cs
using System;

public class URI {
 
    public static void Main(string[] args){
 
        int NUMBER = Int32.Parse( System.Console.ReadLine().Trim() );
        int HOURS = Int32.Parse( System.Console.ReadLine().Trim() );
        double HOUR_VALUE = Convert.ToDouble( System.Console.ReadLine().Trim() );
        double SALARY = HOURS * HOUR_VALUE;

        Console.Write("NUMBER = {0}\n", NUMBER);
        Console.Write("SALARY = U$ {0:F2}\n", SALARY);
 
    }
 
}

File: uri/1 - Basics/1008/Main.go
package main

import "fmt"

func main() {

	var NUMBER, HOURS int
	var HOUR_VALUE float64
	fmt.Scanf("%d", &NUMBER)
	fmt.Scanf("%d", &HOURS)
	fmt.Scanf("%f", &HOUR_VALUE)
	SALARY := float64(HOURS) * HOUR_VALUE

	fmt.Printf("NUMBER = %d\n", NUMBER)
	fmt.Printf("SALARY = U$ %.2f\n", SALARY)
}
